x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -6,11 +6,13 @@
  */
 
 import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { groupBy } from 'lodash';
 import { schema } from '@kbn/config-schema';
 import type { ErrorType } from '@kbn/ml-error-utils';
 import type { CloudSetup } from '@kbn/cloud-plugin/server';
 import type { ElserVersion } from '@kbn/ml-trained-models-utils';
 import { isDefined } from '@kbn/ml-is-defined';
+import { IScopedClusterClient } from '@kbn/core-elasticsearch-server';
```

Comment:
```suggestion
import type { IScopedClusterClient } from '@kbn/core-elasticsearch-server';
```

Reply:
Updated in https://github.com/elastic/kibana/pull/175880/commits/78ef95e7683611d1c5418c81e7de18507453f421